### ISSUE TITLE
[xcode11] Managed filters not found for CIDocumentEnhancer, CIKMeans, CIMorphologyRectangleMaximum, CIMorphologyRectangleMinimum, CIPaletteCentroid, CIPalettize

### ISSUE THREAD

**Issue by spouliot** [xamarin/xamarin-macios#6263]
### Steps to Reproduce

1. Run introspection tests on iOS

### Expected Behavior

No error, including no missing CoreImage filters

### Actual Behavior

Two of the failures are canaries to detect new/missing CoreImage filters. 

Filters are not shown in the API diffs, nor in Xcode headers.

```
2019-06-11 09:01:26.077370-0400 introspection[30897:28286550] 	[FAIL] iOSCoreImageFiltersTest.ApiCoreImageFiltersTest.CheckManagedFilters :   Managed filters not found for CIDocumentEnhancer, CIKMeans, CIMorphologyRectangleMaximum, CIMorphologyRectangleMinimum, CIPaletteCentroid, CIPalettize
  Expected: 0
  But was:  6
2019-06-11 09:01:26.077543-0400 introspection[30897:28286550] 
2019-06-11 09:01:26.077661-0400 introspection[30897:28286550] 		  at Introspection.ApiCoreImageFiltersTest.CheckManagedFilters () [0x0019e] in /Users/poupou/git/d16-1/xamarin-macios/tests/introspection/ApiCoreImageFiltersTest.cs:146
2019-06-11 09:01:26.077753-0400 introspection[30897:28286550] 		  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
2019-06-11 09:01:26.077837-0400 introspection[30897:28286550] 		  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0003b] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.10.0.155/src/Xamarin.iOS/mcs/class/corlib/System.Reflection/MonoMethod.cs:305
2019-06-11 09:01:26.096449-0400 introspection[30897:28286550] 	[FAIL] iOSCoreImageFiltersTest.ApiCoreImageFiltersTest.CheckNativeFilters :   6 native filters missing: CIDocumentEnhancer, CIKMeans, CIMorphologyRectangleMaximum, CIMorphologyRectangleMinimum, CIPaletteCentroid, CIPalettize
  Expected: 0
  But was:  6
2019-06-11 09:01:26.096570-0400 introspection[30897:28286550] 
2019-06-11 09:01:26.096653-0400 introspection[30897:28286550] 		  at Introspection.ApiCoreImageFiltersTest.CheckNativeFilters () [0x00079] in /Users/poupou/git/d16-1/xamarin-macios/tests/introspection/ApiCoreImageFiltersTest.cs:96
2019-06-11 09:01:26.096741-0400 introspection[30897:28286550] 		  at (wrapper managed-to-native) System.Reflection.MonoMethod.InternalInvoke(System.Reflection.MonoMethod,object,object[],System.Exception&)
2019-06-11 09:01:26.096844-0400 introspection[30897:28286550] 		  at System.Reflection.MonoMethod.Invoke (System.Object obj, System.Reflection.BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x0003b] in /Library/Frameworks/Xamarin.iOS.framework/Versions/12.10.0.155/src/Xamarin.iOS/mcs/class/corlib/System.Reflection/MonoMethod.cs:305
```

### Environment

* d16-1 XI
* Xcode 11 beta 1

**Comment by spouliot** [comment]
https://github.com/xamarin/xamarin-macios/pull/6204/commits/5206d3d4ba9023cd5b7848dba591c0e38c168f04 has the missing filters

**Comment by spouliot** [comment]
Fixed in #6204 merge
